pretext/ExtraTopics/None.ptx

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-none">
+  <title>None</title>
+
+  <p><c>None</c>, or the <term>NoneType</term>, is a type which represents the <em>absence</em> of a value.
+    But let's take a step back and think: why is this useful?</p>
+
+  <p>In Python, sometimes we don't want to return anything from a function. However, Python implicitly will return None 
```

Comment:
However, when we don't explicitly return anything from a function, Python will implicitly return None.